content/curating/translation.md

```diff
@@ -0,0 +1,93 @@
+- **Verifying** if the translation holds to [translation authoring guidelines][translation-authoring-guidelines].
+
+After a review is completed, the reviewer can finalize it with one of the following actions:
+
+- **Leaving a comment** with remarks, suggestions and issues which should be addressed by the translator.
+- **Fixing problems found during the review**. Only the author of the translation can edit its fork directly. If a user wants to make changes to a translation fork created by someone else, they have to fork it first, edit the code, and publish the changes as a new fork.
+- **Approving the translation**, which introduces a new language to the kata available for solving.
+- **Rejecting the translation** if it cannot be fixed.
+
+Users performing a review should hold to [Translation Curating Guidelines][translation-curating-guidelines].
+
+### Review comments
+
+When a reviewer writes a post pointing out the spotted issues which need to be fixed, it should be done in a way which actually instructs the translator how to fix them. Sometimes, authors can be not aware that some specific issue might occur, or why this might be a problem, and reviewers with their short remarks leave them even more clueless. When a reviewer wants to signal such issue, it should be done in a way which would minimize the need for follow-up questions and additional explanations. Initially it might seem as more effort for reviewer, but this additional effort might be finally worth it if it helps to resolve the problems easier.
+
+### Approving a translation
+
+Pending translations can be approved by either the kata author at any time, or any user with the [privilege to coauthor][privileges-reference] under the following conditions:
+
+- the approver is not the author of the translation, and
+  - translation is older than a week, or
+  - kata author has been inactive for over a month.
+
+When the translation is approved, a new language is immediately available for solving. There's also an honor reward, but remember that it's granted to the author of the **initial fork** and not that of the approved fork!
+
+The user who approved the translation should also go through the kata discourse and mark suggestions posted by the translator as resolved.
+
+**NOTE:** at the time of writing, a user who _approves_ a translation is marked as a **kata contributor**. Users with the "contributor" flag seem to be affected by a bug preventing them from earning any future points they would be rewarded for authoring activities on kata they contributed to. Be aware that as a result, approving a translation will render you ineligible for honor points you would otherwise earn by translating, editing or fixing the kata in the future unless the bug is fixed.
```

Comment:
```suggestion
**NOTE:** at the time of writing, a user who _approves_ a translation is marked as a **kata contributor**. Users with the "contributor" flag seem to be affected by a bug preventing them from earning any future points they would be rewarded for authoring activities on kata they contributed to. Be aware that as a result, approving a translation will render you ineligible for honor points you would otherwise earn by translating, editing or fixing the kata in the future until the bug is fixed.
```